### DISCUSSION TITLE
How can I access dashboards in other organizations anonymously? It's not on main.org

### DISCUSSION THREAD

**Discussion by kwenzh** [grafana/grafana#73919] [Alerting]
i had settting grafana config
```
[auth.anonymous]
enabled = true
org_name = Main Org.
org_role = Viewer
```
dashboard url about main.org, orgId=1， it is ok.

but can not access dashboard in other org anonymously,  such as : orgid=4，5，6， it will be redirct to login  page。

Is there any good way to solve this? The apikey is known， setting token in header : Authorization "Bearer eyJrIjoixxxx'

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
